index.html

```diff
@@ -4669,105 +4714,130 @@ <h3>Data First Approaches</h3>
+"statusReason": {
+  "value": "<span class="highlight">Revoked</span>",
+  "lang: "<code>en</code>"
+}
+      </pre>
+
       <p>
-[[!JSON-LD]] supports the expression of text layout information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
-<a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
-This mechanism enables developers to use the internationalization features
-present in HTML to express language information. For example, consider the
-following key-value pair:
-
-      <pre>
-"nameHtml": "&lt;span dir="rtl" lang="ar">HTML و CSS: تصميم و إنشاء مواقع الويب&lt;/span>"
+The next example expresses the English text "revoked" in the Arabic
+language with a base direction of right-to-left:
+      </p>
+
+      <pre class="example nohighlight" title="Arabic text with a base direction of right-to-left">
+"statusReason": {
+  "value": "<span class="highlight">إلغاء</span>",
+  "lang: "<code>ar</code>"
+  "dir": "<code>rtl</code>"
+}
       </pre>
 
       <p>
-If the JSON-LD Context states that any value associated with the
-<code>nameHtml</code> property is of type <code>rdf:HTML</code>, then software
-agents have sufficient information to deterministically identify the text
-direction of the language.
+Utilization of the design pattern above assumes that the JSON-LD
+<a href="#extensibility">extension</a> associated with the
+<a>property</a> utilizes the Scoped Context feature of JSON-LD to
+alias the <code>@value</code>, <code>@language</code>, and
+<code>@direction</code> keywords to <code>value</code>, <code>lang</code>,
+and <code>dir</code>, respectively. An example of a JSON-LD Context snippet
+doing so is shown below:
       </p>
 
+      <pre class="example nohighlight" title="Specifying scoped aliasing for language information">
+"statusReason": {
+  <span class="highlight">"@context": {"value": "@value", "lang": "@language", "dir": "@direction"}</span>,
+  "@id": "https://www.w3.org/2018/credentials/examples#statusReason"
+}
+      </pre>
+
     </section>
 
     <section class="informative">
-      <h3>Ruby Text</h3>
+      <h3>Complex Language Markup</h3>
+
       <p>
-In some situations it is common to use
-<a href="https://en.wikipedia.org/wiki/Ruby_character">Ruby characters</a>
-to ensure text information is clear and as useful as possible.
-[[!JSON-LD]] support for HTML data allows text to be encoded with ruby,
-supporting these cases.
+When multiple languages, base directions, and/or annotations are used in a
+single natural language string, more complex mechanisms are typically required.
+It is possible to utilize markup languages, such as HTML, to encode
+text with multiple language and/or base directions. It is also
+possible to use the <code>rdf:HTML</code> datatype to encode such values
+accurately in JSON-LD.
+      </p>
+
+      <p>
+Implementers are strongly discouraged from encoding information as HTML
```

Comment:
`Implementers are`
-> `Despite that possibility, implementers are`

Reply:
Fixed in 966b34891aba8de46db1b8c1a4e8912ec905b8a7.